qiskit_experiments/exceptions.py

```diff
@@ -0,0 +1,19 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Exceptions for errors raised by Qiskit Experiments."""
+
+from qiskit.exceptions import QiskitError
+
+
+class AnalysisError(QiskitError):
```

Comment:
No general `ExperimentError`? For example qiskit nature has the general qiskit nature error:
https://github.com/Qiskit/qiskit-nature/blob/main/qiskit_nature/exceptions.py

Reply:
I thought the consensus last meeting was to just use QiskitError.